spec/customer_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'customer'
+
+describe Customer do
+  let (:subject) {Customer.new{Takeaway.new}}
+
+  it{is_expected.to respond_to(:check_menu)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.